### ISSUE TITLE
Server returned 401 Unauthorized

### ISSUE THREAD

**Issue by glassbase** [elad-bar/ha-blueiris#29]
I get `Server returned 401 Unauthorized (authorization failed): 'http://10.0.0.7:8080/h264/BackYard/temp.m3u8'` when I stream the camera feed from HA.

The integration sets up fine and creates everything. I just can't stream my cameras in the HA interface. The camera picture refreshes ok. 

Any suggestions?

**Comment by glassbase** [comment]
I had my BI cameras working like this. I have to have the username:password in URL for it to work. I noticed there is an open PR for this.

```
camera:
- platform: generic
  name: Back Yard
  still_image_url: "http://homeassistant:homeassistant@10.0.0.7:8080/image/backyard"
  stream_source: "http://homeassistant:homeassistant@10.0.0.7:8080/h264/BackYard/temp.m3u8"
```

**Comment by glassbase** [comment]
For anyone else that has this issue. Here is how I fixed it.

Web server - remote / external access - UNCHECK Stunnel is installed for HTTPS port. I must have enabled that when I was doing configuration to use the Blue Iris app outside my home thinking it was required. 

(I had been wondering why all guides for Home Assistant and Homebridge, etc said it should work with auth, but I had to include usernmae:password. Now I don't)

Cheers
